samples/server/go-server/api/server/api/api.go

```diff
@@ -29,16 +29,18 @@ import (
 	server "github.com/grafeas/grafeas/server-go"
 	pb "github.com/grafeas/grafeas/v1alpha1/proto"
 	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/rs/cors"
 	opspb "google.golang.org/genproto/googleapis/longrunning"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
 )
 
 type Config struct {
-	Address  string `yaml:"address"`  // Endpoint address, e.g. localhost:10000
-	CertFile string `yaml:"certfile"` // A PEM eoncoded certificate file
-	KeyFile  string `yaml:"keyfile"`  // A PEM encoded private key file
-	CAFile   string `yaml:"cafile"`   // A PEM eoncoded CA's certificate file
+	Address            string   `yaml:"address"`              // Endpoint address, e.g. localhost:10000
+	CertFile           string   `yaml:"certfile"`             // A PEM eoncoded certificate file
+	KeyFile            string   `yaml:"keyfile"`              // A PEM encoded private key file
+	CAFile             string   `yaml:"cafile"`               // A PEM eoncoded CA's certificate file
+	CORSAllowedOrigins []string `yaml:"cors_allowed_origins"` // Permitted CORS origins defaults to "*"
```

Comment:
Your comment indicates that this defaults to all origins? If that's the case, we definitely don't want that, prefer default to empty.

Reply:
Yes, I'll update the docs and change that comment as it defaults closed.